libs/ledger-live-common/src/families/celo/types.ts

```diff
@@ -1,20 +1,99 @@
 import type {
+  Account,
+  AccountRaw,
   TransactionCommon,
   TransactionCommonRaw,
   TransactionStatusCommon,
   TransactionStatusCommonRaw,
 } from "@ledgerhq/types-live";
 import { BigNumber } from "bignumber.js";
 
+export type CeloOperationMode =
+  | "send"
+  | "lock"
+  | "unlock"
+  | "withdraw"
+  | "vote"
+  | "revoke"
+  | "activate"
+  | "register";
+
+export type CeloVoteType = "pending" | "active";
+export type CeloVoteStatus = CeloVoteType | "awaitingActivation";
+
+export type CeloPendingWithdrawal = {
+  value: BigNumber;
+  time: BigNumber;
+  index: number;
+};
+export type CeloPendingWithdrawalRaw = {
+  value: string;
+  time: string;
+  index: string;
+};
+export type CeloResources = {
+  registrationStatus: boolean;
+  lockedBalance: BigNumber;
+  nonvotingLockedBalance: BigNumber;
+  pendingWithdrawals: CeloPendingWithdrawal[] | null | undefined;
+  votes: CeloVote[] | null | undefined;
+  electionAddress: string | null | undefined;
+  lockedGoldAddress: string | null | undefined;
+};
+export type CeloResourcesRaw = {
+  registrationStatus: boolean;
+  lockedBalance: string;
+  nonvotingLockedBalance: string;
+  pendingWithdrawals: CeloPendingWithdrawalRaw[] | null | undefined;
+  votes: CeloVoteRaw[] | null | undefined;
+  electionAddress: string | null | undefined;
+  lockedGoldAddress: string | null | undefined;
+};
 export type Transaction = TransactionCommon & {
   family: "celo";
   fees: BigNumber | null | undefined;
+  mode: CeloOperationMode;
+  index: number | null | undefined;
+  address: string | null | undefined;
+  group: string | null | undefined;
```

Comment:
_address_ and _group_ are very vague, I suggest more specific field names, just to ease readability and understanding.

Reply:
As a matter of fact, I was able to remove these - they were unnecessary.